packages/components/src/internal/components/domainproperties/models.tsx

```diff
@@ -283,6 +303,111 @@ export class DomainDesign
 
         return mapping;
     }
+
+    getGridData(): List<any> {
+        return this.fields.map((field, i) => {
+            const fieldSerial = DomainField.serialize(field);
+
+            // Add field properties stripped by the serialize
+            fieldSerial['fieldIndex'] = i;
+            fieldSerial['selected'] = field.selected;
+            fieldSerial['visible'] = field.visible;
+
+            return Map(
+                Object.keys(fieldSerial).map(key => {
+                    const rawVal = fieldSerial[key];
+                    const valueType = typeof rawVal;
+                    let value = valueIsEmpty(rawVal) ? '' : rawVal;
+
+                    // Make bools render as strings sortable within their column
+                    if (key !== 'visible' && key !== 'selected' && valueType === 'boolean') {
+                        value = rawVal ? 'true' : 'false';
+                    }
+
+                    // Handle conditional format rendering
+                    if (key === 'conditionalFormats' && value !== '') {
+                        value = JSON.stringify(value.map(cf => removeFalseyObjKeys(cf)));
+                    }
+
+                    // Handle property validator rendering
+                    if (key === 'propertyValidators' && value !== '') {
```

Comment:
can this be collapsed with the if statement above for conditionalFormats? looks like they both use the same statement within the if block

Reply:
Aaaaa how did I not catch that. Collapsed!